docs/release_notes/10_0.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Current Release 10
+tags:
+  - 10
+  - 10.0 release
+  - Rocky Linux 10
+---
+
+
+## Rocky Linux 10
+
+Rocky Linux 10 (RL 10) is the latest major release of Rocky Linux's enterprise operating system. 
+
+### New Features and Changes
+
+The following sections describe selected features of RL 10. 
+
+As with all major Rocky Linux releases, there are way too many enhancements, changes, and new features to go over in this release note. 
+
+The overview that follows highlights changes in the -  Architecture, Kernel, Installer, Networking, Security, Virtualization, Containers, Storage, Package management - categories. The features highlighted in this release note have been chosen based on their likely relevance to a general audience. 
+
+For more information, refer to the official upstream [Release Notes for Red Hat Enterprise Linux 10.0](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10/html/10.0_release_notes/index). 
+
+### Supported Architectures
+
+Rocky Linux 10 is supported on the following processor architectures:
+
+* 64-bit AMD/Intel x86-64-v3 (x86_64)  
+* 64-bit RISC-V (riscv64)  
+* 64-bit ARMv8.0-A (aarch64)  
+* ??? IBM POWER9, little endian (ppc64le) ???  
+* ??? IBM z14 (s390x) ???
+
```

Comment:
Something something we do not support upgrades from 8 or 9 🙂

Reply:
Ah yes. This note is in previous release note sets, but not here. I'll add that at the top.